scripts/monorepo/findGitRoot.js

```diff
@@ -0,0 +1,20 @@
+const fs = require('fs');
+const path = require('path');
+
+function findGitRoot() {
```

Comment:
Don't we already have this code or something super similar elsewhere?

Reply:
This was already in just somewhere - not sure if it's exposed though.